rasa/nlu/extractors/crf_entity_extractor.py

```diff
@@ -529,27 +573,64 @@ def __tag_of_token(token):
         else:
             return token.tag_
 
+    @staticmethod
+    def __additional_ner_features(message: Message) -> List[Any]:
+        features = message.get("ner_features", [])
+        tokens = message.get("tokens", [])
+        # perhaps an assertion here instead? following pattern above.
```

Comment:
What do you mean by assertion? cannot find anything like that above? We should either decide to implement it or remove the comment.

Reply:
oh yeah I worded that poorly. I meant "maybe we should have an assertion here, BUT I'm following the pattern above instead". I'll remove the comment if you think it looks okay